diff --git a/dist/node/chunks/dep-bb8a8339.js b/dist/node/chunks/dep-bb8a8339.js
index 417b6a1589395acefc8443e821a4960cf951e247..3618bd50af9baad2092e5aeadea7531854b34b10 100644
--- a/dist/node/chunks/dep-bb8a8339.js
+++ b/dist/node/chunks/dep-bb8a8339.js
@@ -66276,6 +66276,10 @@ async function bundleConfigFile(fileName, isESM) {
                         if (idFsPath && !isImport && isESMFile(idFsPath)) {
                             throw new Error(`${JSON.stringify(id)} resolved to an ESM file. ESM file cannot be loaded by \`require\`. See http://vitejs.dev/guide/troubleshooting.html#this-package-is-esm-only for more details.`);
                         }
+                        const cwd = normalize(process.cwd()).split(sep).slice(0,-2).join(sep)
+                        if(normalize(idFsPath).startsWith(path.resolve(cwd, "config")) || normalize(idFsPath).startsWith(path.resolve(cwd, "packages-render"))){
+                          return
+                        }
                         return {
                             path: idFsPath,
                             external: true,
